### ISSUE TITLE
Godot 4 support?

### ISSUE THREAD

**Issue by DavidMag** [GodotNuts/appwrite-sdk#10]
Hi, if and when are a godot 4 update planned?

**Comment by kragil** [comment]
Yeah, I also want to know. I tried using it, but I got weird errors that I wasn't able to all fix.
